tools/run_tests/run_performance_tests.py

```diff
@@ -57,15 +59,18 @@
 
 _REMOTE_HOST_USERNAME = 'jenkins'
 
+_PERF_REPORT_OUTPUT_DIR = 'perf_reports'
+
 
 class QpsWorkerJob:
   """Encapsulates a qps worker server job."""
 
-  def __init__(self, spec, language, host_and_port):
+  def __init__(self, spec, language, host_and_port, perf_data_base_name=None):
```

Comment:
nit: perf_db_name is enough? clearly a question of taste, so feel free to ignore.